packages/manager/src/components/TableContentWrapper/TableContentWrapper.tsx

```diff
@@ -37,6 +52,26 @@ const TableContentWrapper: React.FC<CombinedProps> = (props) => {
   }
 
   if (lastUpdated !== 0 && length === 0) {
+    if (isAkamaiCustomer) {
+      return (
+        <TableRow>
+          <TableCell colSpan={6}>
+            <Grid
+              container
+              justifyContent="center"
+              alignItems="center"
+              className={classes.grid}
+            >
+              <Grid item>
+                <Typography style={{ textAlign: 'center' }}>
+                  <strong>{akamaiBillingInvoiceText}</strong>
+                </Typography>
+              </Grid>
+            </Grid>
```

Comment:
Seems like this Grid stuff could be simplifed to something like 

```suggestion
            <Box alignItems="center" justifyContent="center">
              <Typography style={{ textAlign: 'center' }}>
                <strong>{akamaiBillingInvoiceText}</strong>
              </Typography>
            </Box>
```